src/scripts/domWidget.ts

```diff
@@ -193,7 +193,9 @@ function computeSize(size: [number, number]): void {
 
 // Override the compute visible nodes function to allow us to hide/show DOM elements when the node goes offscreen
 const elementWidgets = new Set();
+//@ts-ignore
 const computeVisibleNodes = LGraphCanvas.prototype.computeVisibleNodes;
+//@ts-ignore
```

Comment:
The fundamental fix of this error requires large rewrite which is hard to review.